ibm-messaging-mq-cloud-showcase-app/frontend/nginx.conf

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2024 IBM Corp.
+#  Licensed under the Apache License, Version 2.0 (the 'License');
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+ 
+#  http://www.apache.org/licenses/LICENSE-2.0
+ 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri $uri/ /index.html;
+    }
+
+    #Set up reverse proxy to redirect requests to backend
+    location /api/ {
+        proxy_pass http://be:8080/api/;
```

Comment:
Http / Https, Host and port need to be set via the environment variables:

````
const be = process.env.REACT_APP_BE_HOST;
const be_port = process.env.REACT_APP_BE_PORT || "8080";

const be_tls = process.env.REACT_APP_BE_TLS || false;
const HTTP_PROTOCOL = be_tls ? "https://" : "http://";
````

See existing proxy setup : https://github.com/ibm-messaging/mq-dev-patterns/blob/2b66777a8208e149bea3c3e24da435fa7b14ae58/ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/src/setupProxy.js#L19

Reply:
should this code be using `be_port` in stead of 8080 ?